itemcollection-spec/itemcollection-spec.md

```diff
@@ -27,7 +27,18 @@ This object describes a STAC ItemCollection. The fields `type` and `features` ar
 | type       | string | **REQUIRED.** always "FeatureCollection" to provide compatability with GeoJSON  |
 | features   | [Item] | **REQUIRED** a possibly-empty array of Items          |
 | links      | [Link] | an array of Links related to this ItemCollection   |
-          
+| properties      | Properties | an array of Links related to this ItemCollection   |
```

Comment:
Having properties in a FeatureCollection is conflicting with the GeoJSON spec. We should NOT do the same mistake as the OGC and do things that are clearly disallowed in the GeoJSON spec!

Reply:
Ugh. 
```
  The GeoJSON "geometry" and "properties" members define a Feature
   object.  FeatureCollection and Geometry objects, respectively, MUST
   NOT contain a "geometry" or "properties" member.
```
So this brings up the question as to whether the non-GeoJSON attributes we're putting in the `properties` object should go in their own, e.g., `stacProperties`

Reply:
What properties do you think could be in there? I don't see a use case yet. It seems nobody missed it yet? (But whatever name it will be: snake_case please ;-) )

Reply:
snake_case, of course.  Right now, we have the Search extension that will augment the ItemCollection with metadata about search.  One option would be to have the convention that Item/Feature extension attributes should go into `properties` (flat), but ItemCollection/FeatureCollection attributes should go into another object (nested)?

Reply:
I have a hard time understanding this / coming up with examples. But regarding the search extension, could this just be placed on the root level in a separate object?

```
{
  search:pagination: {
    next: ...
  },
  type: "FeatureCollection",
  features: [...], 
  ...
}
```

Something like that?

Reply:
Yes, that's exactly what we'd done with the search extension, before i started down this path of thinking it should be in Properties. I'll just revert to what we had before for search extension.